docs/plans/3853-clickhouse-dictionary-support.md

```
@@ -0,0 +1,1015 @@
+    layout: dict  # serialized layout config
+    lifetime: Union[int, dict]
+    invalidate: Optional[dict] = None
+    defaults: Optional[dict] = None
+    settings: Optional[dict] = None
+    comment: Optional[str] = None
+    metadata: Optional[dict] = None
+    life_cycle: str = "FULLY_MANAGED"
+```
+
+Add to `InfrastructureMap`:
+```python
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)